shared/src/main/kotlin/org/javacs/kt/SourceExclusions.kt

```diff
@@ -13,8 +13,10 @@ class SourceExclusions(
     private val workspaceRoots: Collection<Path>,
     private val scriptsConfig: ScriptsConfiguration
 ) {
-	val excludedPatterns = (listOf(
-        ".*", "bazel-*", "bin", "build", "node_modules", "target"
+    val excludedPatterns = (listOf(
+        ".git", ".hg", ".svn",                                                      // Version control systems
+        ".idea", ".idea_modules", ".vs", ".vscode", ".code-workspace", ".settings", // IDEs
+        "bazel-*", "bin", "build", "node_modules", "target",                        // Build systems
```

Comment:
Hm, I would rather integrate with .gitignore or a custom file (e.g. .klsignore) than adding to this ever-growing list of hardcoded directories... my worry is that directory names like `target` are already so generic, that there might be non-Maven-projects that place their source code there, for example

Reply:
Yeah I agree, in the long term we have got to fix this... but for now this fix shouldn't hurt.